library/src/tensile_host.cpp

```diff
@@ -235,12 +257,18 @@ struct TensileHostImpl : TensileHost
                 path = info.dli_fname;
                 dirname(&path[0]);
                 path.resize(strlen(path.c_str()));
-                path += "/../../Tensile/library";
             }
             else
             {
-                path = "/opt/rocm"; // TODO: Need to set default
+                path = "/opt/rocm/rocblas/lib"; 
             }
+	    // Find the location of the libraries
+            if (TestPath(path + "/../../Tensile/library"))
```

Comment:
This assumes library is built in the same folder as the source. This is not necessarily true.